core/client/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -284,6 +284,7 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     try {
       masterClient.setAttribute(path, options);
+      LOG.info("Set attributes of file {} to ({})", path.getPath(), generateLogInfo(options));
```

Comment:
I think SetAttributeOptions already has a `toString()`, so I don't think you need to add a `generateLogInfo()` method.

Also, I think you can use "Set attributes for path {} with options {}"